File: pyjl/tests/tests_py2many/fstring.py
#!/usr/bin/env python3

if __name__ == "__main__":
    a = 10
    assert f"hello {a+1} world" == "hello 11 world"
    assert f"test {a+1} test {a+20} test {a+30}" == "test 11 test 30 test 40" 
    print("OK")

File: pyjl/tests/tests_py2many-py2many/fstring.jl
function main()
a = 10
@assert(join(["hello ", string(a + 1), " world"], "") == "hello 11 world")
@assert(join(["test ", string(a + 1), " test ", string(a + 20), " test ", string(a + 30)], "") == "test 11 test 30 test 40")
println("OK");
end

main()
